src/pages/Auth/SignUp.jsx

```diff
@@ -1,5 +1,109 @@
-export default function SignUp(){
-    return(
-        <div>SignIn page</div>
-    )
-}
\ No newline at end of file
+import { BiRightArrow } from "react-icons/bi";
+import { Link } from "react-router-dom";
+import { useAuth } from "../../context/AuthContext";
+
+export default function SignUp() {
+
+  const {signup} = useAuth();
+
+  const submit = () =>{
+    signup();
+  }
+  return (
+    <section className="main-container">
+      <div className="signup">
+        <h2 className="heading">Signup</h2>
+        <form action="#" className="signup-form" id="signup" onSubmit={submit}>
+          <div className="input-block-name">
+            <div className="input-block">
+              <label for="#input-name" className="text-align-left">
+                First Name
+              </label>
+              <input
+                type="text"
+                id="input-name"
+                placeholder="First Name"
+                required
+                // onChange={(e) => changeInputHandler(e)}
+              />
+            </div>
+            <div className="input-block">
+              <label for="#input-lastname" className="text-align-left">
+                Last Name
+              </label>
+              <input
+                type="text"
+                id="input-lastname"
+                placeholder="Last Name"
+                required
+                // onChange={(e) => changeInputHandler(e)}
+              />
+            </div>
+          </div>
+          <div className="input-block">
+            <label for="#input-email" className="text-align-left">
+              Email Address
+            </label>
+            <input
+              type="email"
+              id="input-email"
+              placeholder="username@email.com"
+              required
+              //   onChange={(e) => changeInputHandler(e)}
+            />
+          </div>
+          <div className="input-block">
+            <label for="input-pass" className="text-align-left">
+              Password
+            </label>
+            <input
+              type="password"
+              id="input-pass"
+              className="input-password"
+              placeholder="*************"
+              required
+              //   onChange={(e) => changeInputHandler(e)}
```

Comment:
same as above. If you don't need them; add a comment.

Reply:
Ok added